CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Snyk Security Changelog
 
+## [2.12.2]
```

Comment:
The versions seems incorrect based on the latest release (which was a few minutes ago). Could you amend 2.12.1 to be under 2.12.0 actually and this one is 2.12.1 please?